### PR TITLE
2.2.x cherry picks

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

**Comment by alfrunes** [comment]
@tranchitella Can you also add the fix for caching chunked request? (6e9dec3161665abd0cf1648a8136837c49e3631b)

**Comment by tranchitella** [comment]
@alfrunes added, thanks for double-checking and spotting it!

File: common.nginx.conf [hunk 1]
```diff
@@ -23,6 +18,8 @@ if ($origin_valid = 0) {
 	return 400;
 }
 
+proxy_http_version 1.1;
```
Comment (kacf) [inline common.nginx.conf:21]: For this fix in particular, can you add a `changelog: Fix Artifact upload timeout bug`. This was a very visible bug, asked about many times on Mender Hub, so it should have a changelog entry.

Comment (tranchitella) [reply common.nginx.conf:21]: Done: https://github.com/mendersoftware/mender-api-gateway-docker/pull/137/commits/fb411b3d52cde694ac464a65149c36470da28371

**Pull request merged**